codeserver/ubi9-python-3.12/run-nginx.sh

```
@@ -6,28 +6,87 @@ set -e
+
+map \$base_path \$api_kernels_path {
+    "/"     "/api/kernels/";
+    default "\$base_path/api/kernels/";
+}
+
+upstream workbench_server {
+    server localhost:8787 max_fails=0;
+}
+EOF
+
+if [ -z "$NB_PREFIX" ]; then
```

Shellcheck (0.11.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)